website/public/js/services/guideServices.js

```diff
@@ -239,6 +239,7 @@ function($rootScope, User, $timeout, $state) {
     if (!updateFn) return; // only run after user has been wrapped
     watcher(); // deregister watcher
     if (window.env.IS_MOBILE) return; // Don't show tour immediately on mobile devices
+    if (User.user.items.gear.owned['weapon_warrior_0']) return;//If the user has the training sword, then don't show the intro tour
```

Comment:
Also, ultimately, we want the tour to not play if the user has started their account on a mobile device, so it might be better to change the `if(window.env.IS_MOBILE)` block to set the flag that the tour should not play (IE, it has been completed).

@lemoness that is still the behavior you want, right?